test/404s.test.js

```diff
@@ -830,21 +830,21 @@ test('setNotFoundHandler should not suppress duplicated routes checking', t => {
 
   const fastify = Fastify()
 
-  fastify.get('/', function (req, reply) {
-    reply.send({ hello: 'world' })
-  })
-
-  fastify.get('/', function (req, reply) {
-    reply.send({ hello: 'world' })
-  })
-
-  fastify.setNotFoundHandler(function (req, reply) {
-    reply.code(404).send('this was not found')
-  })
+  try {
+    fastify.get('/', function (req, reply) {
+      reply.send({ hello: 'world' })
+    })
+    fastify.get('/', function (req, reply) {
+      reply.send({ hello: 'world' })
+    })
+    fastify.setNotFoundHandler(function (req, reply) {
+      reply.code(404).send('this was not found')
+    })
 
-  fastify.listen(0, err => {
-    t.ok(err)
-  })
+    t.fail('setNotFoundHandler should not interfer duplicated route error')
```

Comment:
"interfer" is not a word I recognize.

Reply:
[This one](https://github.com/fastify/fastify/pull/2954#discussion_r614385800) and here solved in 5bfeeb 👍

Reply:
I'm still seeing this on GitHub.

Reply:
I made another typo, sorry 😓 
Done in 0659eb5